guides/doc-Administering_Project/docinfo.xml

```diff
@@ -1,7 +1,7 @@
 <title>Administering Red Hat Satellite</title>
 <productname>Red Hat Satellite</productname>
 <productnumber>6.14</productnumber>
-<subtitle>A guide to administering Red Hat Satellite.</subtitle>
+<subtitle>Administer Satellite users and permissions, manage organizations and locations, backup and restore Satellite, maintain Satellite, and more</subtitle>
```

Comment:
```suggestion
<subtitle>Administer users and permissions, manage organizations and locations, back up and restore Satellite, maintain Satellite, and more</subtitle>
```

Reply:
Dropping 'Satellite' because the subtitle is very long.
Using 'back up' because it's a verb, not a noun.